grails-forge-core/src/main/java/org/grails/forge/feature/github/workflows/templates/javaSetup.rocker.raw

```diff
@@ -5,9 +5,9 @@
 JdkVersion jdkVersion
 )
 
-      - uses: actions/checkout@@v2
+      - uses: actions/checkout@@v4
       - name: Set up JDK @jdkVersion.majorVersion()
-        uses: actions/setup-java@@v2
+        uses: actions/setup-java@@v4
```

Comment:
Use the latest version of `actions/checkout` and  `actions/setup-java`

https://github.com/actions/checkout/tags
https://github.com/actions/setup-java/tags